finance/community-updates/2022-10.txt

```diff
@@ -0,0 +1,94 @@
+Dear Astropy Community,
+
+The Finance Committee would like to give you some updates on our activities
+over the several months.
+
+
+# Cycle III Funding Results
+
+We recently completed the proposal process for allocating the next round of
+funding for projects and other major expenses. This is the first time we have
+been able to allocate money from our NASA ROSES-2020 grant as well as our
+grant from the Moore foundation.
+
+A total of 22 proposals were received, of which 20 were allocated funds.
+Although our budget didn't quite stretch to supporting all projects at their
+full requested level, every approved proposal receives at least their minimum
+funding request, and — in many cases — substantially more.
```

Comment:
Add something about "we went based on the community's votes"?

Reply:
I feel like that is asking for somebody to find a counter-example or otherwise quibble!